documentation/client.md

```diff
@@ -431,6 +434,57 @@ When the scanner successfully decodes a `TransactionRequest` it will call its de
 
 You should use this `TransactionRequest` to generate a `TransactionConsumptionParams` in order to [consume the request](#consume-a-transaction-request).
 
+### Reset user password
+
+> Note: Only available if the eWallet is running in standalone mode.
+
+If a user forget his password, he will be able to reset it by requesting  a reset password link.
+This link will contain a unique token that will need to be submitted along with the updated password chosen by the user.
+
+#### Reset password
+
+To get a password reset link, you can call:
+
+```
+User.resetPassword(using: client, params: params) { result in
+    switch result {
+    case .success:
+      // Handle success
+    case let .fail(error: error):
+      // Handle error
+    }
+}
+```
+
+Where `params` is a `UserResetPasswordParams` struct constructed using:
+
+- `email`: The email of the user (ie: email@example.com)
+- `redirectURL`: A redirect URL that will be built on the server by replacing the `{email}` and `{token}` params.
+
+For example, if you provide this url: `my-app-scheme-uri://user/reset_password?email={email}&token={token}`, the user will receive a link by email that will look like this: `my-app-scheme-uri://user/reset_password?email=email@example.com&token=XXXXXXXXXXXXXXXXXXXXXX`.
+You can then handle the params passed to your application upon launch from this deep link and pass them to the `User.updatePassword` method.
+
```

Comment:
I think it would be nice if we mention the `redirectURL` prefix need to be supported by eWallet, or else we will get the error `The given redirect_url is not allowed.`

Reply:
👍